src/toolkit.cpp

```diff
@@ -518,6 +527,26 @@ bool Toolkit::LoadData(const std::string &data)
     return this->LoadData(data, true);
 }
 
+void Toolkit::SetViewAndEditor()
+{
+    m_view.SetDoc(&m_doc);
+
+#if defined NO_HUMDRUM_SUPPORT
```

Comment:
Just for my understanding: why is the editor toolkit only set, if we have no humdrum support?

Reply:
I am not sure either. Maybe there was some dependencies at some point. I have [introduced](https://github.com/rism-digital/verovio/pull/4045) a new define for the editor. I'll have a look and make this consistent.